mbedtls/src/x509/certificate.rs

```diff
@@ -180,6 +189,26 @@ impl LinkedCertificate {
         unsafe { ::core::slice::from_raw_parts(self.inner.raw.p, self.inner.raw.len) }
     }
 
+    pub fn version(&self) -> i32 {
+        self.inner.version
+    }
+
```

Comment:
Might this be better as an enum?

Reply:
enum seems fine and is probably more convenient, will change